### PR TITLE
add framework search path to support building xcframeworks via Carthage

### PR DESCRIPTION
**Pull request by kazedayo**

### CODE REVIEW

**Comment by r-plus** [comment]
i just try your branch to build xcframeworks via carthage, but still failing compilation.

**Comment by r-plus** [comment]
Wow, now buildable 👍

**Comment by r-plus** [comment]
hmm, looks like after stepping down of @sunshinejr, this repository have not active maintainer...?

**Comment by AndrewSB** [comment]
I'll take a look at this today. thanks for your help @r-plus!

File: Moya.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -963,20 +963,24 @@
...
 				"FRAMEWORK_SEARCH_PATHS[sdk=appletv*]" = (
-					"$(SRCROOT)/Carthage/Build/tvOS/",
 					"$(inherited)",
+					"$(SRCROOT)/Carthage/Build/",
+					"$(SRCROOT)/Carthage/Build/tvOS",
 				);
 				"FRAMEWORK_SEARCH_PATHS[sdk=iphone*]" = (
-					"$(SRCROOT)/Carthage/Build/iOS/",
 					"$(inherited)",
+					"$(SRCROOT)/Carthage/Build/",
```
Comment (AndrewSB) [inline Moya.xcodeproj/project.pbxproj:972]: why did you have to add both this line and `"$(SRCROOT)/Carthage/Build/iOS"`?

Comment (AndrewSB) [reply Moya.xcodeproj/project.pbxproj:972]: not sure if you did this on purpose, or if it's a default from Xcode, but maybe you can remove this line? looks like it happened for every scheme other than the tvOS one

Comment (kazedayo) [reply Moya.xcodeproj/project.pbxproj:972]: I did it on purpose to maintain compatibility for building old style framework bundle.

I can try for a more elegant solution if possible

Comment (AndrewSB) [reply Moya.xcodeproj/project.pbxproj:972]: ahh that makes sense. no that's fine then, didn't realize that was the old style.iI've seen it before, i thought it was some Xcode default behavior. will merge after i test

**Comment by AndrewSB** [comment]
testing this on an M1 machine, will report back in a few hours

**Comment by AndrewSB** [comment]
looks like it builds with carthage. i appreciate all of the work you put into this 😄

**Comment by r-plus** [comment]
Great!
I want to use this change as v14.1.0 or v14.0.1 release.

**Comment by laxmorek** [comment]
@kazedayo Thanks for providing this fix :) 

@AndrewSB Do you have plans to release this fix as patch of 14.x or as 15.0? Are you able to give an estimated date of release? :)

**Comment by AndrewSB** [comment]
happy to do it as a patch on 14.x, I don't have enough context to do a major release at the moment. I'll cut the patch release today

**Comment by r-plus** [comment]
thanks for releasing.

maybe missing update of podfile version but it is required for cocoapods users, v14.0.1 changes are only required carthage users so no problem 👍

**Comment by laxmorek** [comment]
@AndrewSB I appreciate for quick response and release. You are the best 🙂

**Comment by AndrewSB** [comment]
@r-plus: I haven't used cocoapods in a few years, and i'm struggling to get my user account setup. I'll let the next contributor take this on when we release 14.0.2/15.0.0.

you're welcome @laxmorek!

**Pull request merged**